database/seed.js

```diff
@@ -27,25 +27,25 @@ const questionSchema = new mongoose.Schema({
   answers: [answerSchema], // subdoc
 });
 
-// GENERATE FAKE DATA
+// let productSchema = new mongoose.Schema({
+//   product_id: String,
+//   user: [questionSchema],
+// })
 
 const genAnswers = (min = 1, max) => {
   const answers = [];
 
-  const rnd = Math.floor(Math.random() * (max - min + 1) + min);
-  for (let n = 0; n < rnd; n += 1) {
+  var rnd = Math.floor(Math.random() * (max - min + 1) + min);
```

Comment:
I wonder why you change to var from const.

Reply:
probably part of my git problems. I now have old code in a few places.